CONTRIBUTING.md

```diff
@@ -61,6 +61,10 @@ To update the component API documentation (auto generated from component PropTyp
 yarn docs:api
 ```
 
+### Building locally
+To use the provided build scripts with yarn you have to install `yarn@^1.4.0`. Depending on the package you want to build just run 
```

Comment:
why v1.4.0?

Reply:
yarnpkg/yarn#4848 was released with 1.4 and 1.3.2 was used in the linked issues. Netlify has no problems because it only runs scripts that are defined in the root package.json.